tests/parser/UDQTests.cpp

```diff
@@ -112,6 +112,7 @@ BOOST_AUTO_TEST_CASE(UDQWellSetTest) {
     for (std::size_t i=0; i < wells.size(); i++)
         BOOST_CHECK_EQUAL(ws3[wells[i]].value(), i*1.0);
 }
+<<<<<<< 337d73fb20ad410d1d0baa28d1cd452ac4f7e60e
```

Comment:
hmmmm

Reply:
:astonished:  - works on my machine!

Reply:
yes, it's removed in 334b985. drop 62bd2d6 and deal with the "conflict" in 334b985